### PR TITLE
add release-notes for grpc stats

### PR DESCRIPTION
**Pull request by zirain**
**Please provide a description of this PR:**
fixes: #43908
add release-notes for https://github.com/istio/proxy/pull/4547

**To help us figure out who should review this PR, please put an X in all the areas that this PR affects.**

- [ ] Ambient
- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [x] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

**Please check any characteristics that apply to this pull request.**

- [ ] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

**Review by hanxiaop — APPROVED** [review]

**Comment by zirain** [comment]
/hold 

need backport to v1.17, wait for https://github.com/istio/proxy/pull/4550

File: releasenotes/notes/grpc-stats.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: telemetry
+issue:
+  - 43908
+releaseNotes:
+  - |
+    **Fixed** an issue that grpc stats are absent.
```
Comment (ericvn) [inline releasenotes/notes/grpc-stats.yaml:8]:
```suggestion
    **Fixed** an issue where grpc stats are absent.
```

File: releasenotes/notes/grpc-stats.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: telemetry
+issue:
+  - 43908
```
Comment (ericvn) [inline releasenotes/notes/grpc-stats.yaml:5]: Is https://github.com/istio/istio/issues/44144 another reference or a better reference?

**Review by ericvn — APPROVED** [review]
Release notes look good.

**Comment by ericvn** [comment]
Remove DNM since https://github.com/istio/proxy/pull/4550 has merged since @johnma14 asked.

**Pull request merged**